### PR TITLE
849 programme page field changes

### PR DESCRIPTION
**Pull request by helenb**

### CODE REVIEW

**Comment by helenb** [comment]
Hi Karl please check but don't actually merge as agreed. Thanks!

File: django-verdant/rca/models.py [hunk 1]
```diff
@@ -821,35 +806,26 @@ class ProgrammePageAd(Orderable):
...
-class ProgrammePageProgramme(models.Model):
-    page = ParentalKey('rca.ProgrammePage', related_name='programmes')
-    programme = models.ForeignKey('taxonomy.Programme', null=True, on_delete=models.SET_NULL, related_name='programme_pages', help_text=help_text('rca.ProgrammePageProgramme', 'programme'))
-
-    panels = [FieldPanel('programme')]
-
-
 class ProgrammePage(Page, SocialFields, SidebarBehaviourFields):
-    school = models.ForeignKey('taxonomy.School', null=True, on_delete=models.SET_NULL, related_name='programme_pages', help_text=help_text('rca.ProgrammePage', 'school'))
+    school = models.ForeignKey('taxonomy.School', null=True, on_delete=models.SET_NULL, related_name='programme_pages')
```
Comment (kaedroho) [inline django-verdant/rca/models.py]: We should keep the ``help_text`` attribute as this makes it easier for us to add help text later.

File: django-verdant/rca/migrations/0064_auto_20170124_0935.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                ('sort_order', models.IntegerField(blank=True, editable=False, null=True)),
+                ('link_text', models.CharField(blank=True, help_text=b'', max_length=255)),
+                ('link', models.ForeignKey(blank=True, help_text=b'', null=True, on_delete=django.db.models.deletion.CASCADE, related_name='+', to='rca.StandardPage')),
+            ],
+            options={
+                'ordering': ['sort_order'],
+                'abstract': False,
+            },
+        ),
+        migrations.AddField(
```
Comment (kaedroho) [inline django-verdant/rca/migrations/0064_auto_20170124_0935.py]: This field was just deleted in https://github.com/torchbox/verdant-rca/pull/98/files#diff-1a0a34590e830aaede2bf11658679d04R19. Do we need to keep the previous content?

File: django-verdant/rca/migrations/0053_auto_20170120_1212.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```
Comment (kaedroho) [inline django-verdant/rca/migrations/0053_auto_20170120_1212.py]: We should delete this migration file

File: django-verdant/rca/migrations/0054_auto_20170120_1221.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+# Generated by Django 1.9.9 on 2017-01-20 12:21
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rca', '0053_auto_20170120_1212'),
```
Comment (kaedroho) [inline django-verdant/rca/migrations/0054_auto_20170120_1221.py]: Update this to ``0052_auto_20170120_1206`` when ``django-verdant/rca/migrations/0053_auto_20170120_1212.py`` is deleted

File: django-verdant/rca/models.py (hunk 1 above)
Comment (helenb) [inline django-verdant/rca/models.py]: I'll re-add those.

File: django-verdant/rca/migrations/0064_auto_20170124_0935.py (hunk 2 above)
Comment (helenb) [inline django-verdant/rca/migrations/0064_auto_20170124_0935.py]: Natalie said we didn't need to worry.

File: django-verdant/rca/migrations/0053_auto_20170120_1212.py (hunk 3 above)
Comment (helenb) [inline django-verdant/rca/migrations/0053_auto_20170120_1212.py]: OK thanks - I did wonder if that should be deleted.

File: django-verdant/rca/migrations/0054_auto_20170120_1221.py (hunk 4 above)
Comment (helenb) [inline django-verdant/rca/migrations/0054_auto_20170120_1221.py]: will do.

**Pull request merged**